### PR TITLE
Don't show state column on import yaml dialog

### LINKED ISSUE #10656: Don't show state column on import yaml dialog
Internal Reference: SURE-7870

After you import YAML, we should a list of the resources that were imported - this includes the state column, e.g.:

image

The state column is not live, so it does not update - this can be confusing/misleading.

We should remove this column as it would be difficult/non-performant to do so.

Note also, in some cases, we will see a sub-row saying something like `Deployment generation is 2, but latest observed generation is 1` - we should not show sub-rows in this view, since this represents an intermediate state and the view is not auto-updating, so this is mis-leading.

### PR DESCRIPTION
**Pull request by aalves08**
### Summary
Fixes #10656 

### Occurred changes and/or fixed issues

- remove state col and subrows from import YAML action success screen 
- prevent sub-row form showing state description
- add tests

### Areas or cases that should be tested

- got header -> Import a YAML -> fill out the YAML editor and hit save (should not show state col)

### Screenshot/Video

https://github.com/rancher/dashboard/assets/97888974/8d9e5972-ed8e-4182-aa17-6fd1da55c28b

### Checklist
- [x] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [x] The PR has a Milestone 
- [x] The PR template has been filled out
- [x] The PR has been self reviewed 
- [x] The PR has a reviewer assigned
- [x] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [x] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes

### CODE REVIEW

File: shell/components/Import.vue [hunk 1]
```diff
@@ -98,7 +95,22 @@ export default {
 
         btnCb(true);
 
-        this.rows = res;
+        const parsedRes = [];
+
+        // cleanup metadata.state and stateDescription to not show sub-rows in this table of results
```
Comment (richard-cox) [inline shell/components/Import.vue:100]: This is a bit hacky, and there's a `subRows` arg on sortable table that should do the job

**Review by richard-cox — CHANGES_REQUESTED** [review]

File: shell/components/Import.vue (hunk 1 above)
Comment (aalves08) [reply shell/components/Import.vue:100]: what do you mean @richard-cox ? is there another way of doing this, especially without modifying the `sortableTable` component?

Comment (richard-cox) [reply shell/components/Import.vue:100]: https://github.com/rancher/dashboard/blob/master/shell/components/SortableTable/index.vue#L161

Comment (aalves08) [reply shell/components/Import.vue:100]: https://github.com/rancher/dashboard/blob/master/shell/components/Import.vue#L156-L166

sub-rows is not defined, therefore it's `false`. Nevertheless, we've got this:
https://github.com/rancher/dashboard/blob/master/shell/components/SortableTable/index.vue#L880-L885

Which means if there's a `stateDescription`, it's always shown

Comment (richard-cox) [reply shell/components/Import.vue:100]: looks like `subRows` is for manually adding a sub row to present something other than description.

we should add a `subRowDescription` param, defaulted to true, which works on https://github.com/rancher/dashboard/blob/master/shell/components/SortableTable/index.vue#L882

**Review by richard-cox — CHANGES_REQUESTED** [review]

**Review by richard-cox — APPROVED** [review]

**Pull request merged**